.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "2.7"
+  - "3.7"
```

Comment:
Should we also run against Python 3.8? Have a matrix of supported Python versions?
Or does this just setup the tox script, in which case we should change that to run against 3.8 too?